config.toml.example

```diff
@@ -48,7 +48,21 @@
 
 	[pageContext.youtube]
 		APIKey = "AIzaSyCzI5Gvpq1hPlfjvB81EVc0vLv_11eW7NI"
-		sessionsPlaylistID = "PLsxUDJzU4VKDudlmlYb-ImFKreFKAfFln"
+		homepagePlaylistID = "PLsxUDJzU4VKDag-NVag4kMN1i5C7ZyP85"
+		homepageDescription = "Fresh music from fresh artists, something witty here..."
+		homepageTitle = "Q-Fest"
+		homepageBGColour = "#235660"
+		homepageColor = "#fff"
+		od1PlaylistID = "PLsxUDJzU4VKDudlmlYb-ImFKreFKAfFln"
```

Comment:
Eeeeeeek. Have you considered just using an array?

Reply:
indeed, there must be a better way. Or maybe toml isnt appropriate for this?

Reply:
https://toml.io/en/v1.0.0-rc.2#array-of-tables looks like you can do something with tables and arrays :)